src/main.rs

```diff
@@ -190,14 +201,16 @@ struct Args {
     #[clap(
         long = "oci-user",
         env = "OCI_REGISTRY_USER",
+        alias = "WASMCLOUD_OCI_REGISTRY_USER",
```

Comment:
Any reason you didn't go for `WASMCLOUD_*` being the `env` value here like other places?

Reply:
good catch! not in particular, glossed over this thanks